docs/source/dataanalysis/dataanalysis.rst

```diff
@@ -69,13 +70,13 @@ The code writes out the values in an ASCII file prefixed ``reduced_beam_characte
 * ``s``, ``ref_beta_gamma``
     Reference particle coordinate ``s`` (unit: meter) and relativistic momentum normalized by the particle mass and the speed of light (unit: dimensionless)
 * ``x_mean/min/max``, ``y_mean/min/max``, ``t_mean/min/max``
-    Average / minimum / maximum beam particle position in the dimensions of ``x``, ``y`` (transverse coordinates, unit: meter), and ``t`` (normalized time difference :math:`ct`, unit: meter)
+    Average / minimum / maximum beam particle displacement with respect to the reference particle in the dimensions of ``x``, ``y`` (transverse coordinates, unit: meter), and ``t`` (normalized time difference :math:`ct`, unit: meter)
 * ``sig_x``, ``sig_y``, ``sig_t``
     RMS of the average beam particle positions (unit: meter)
 * ``px_mean/min/max``, ``py_mean/min/max``, ``pt_mean/min/max``
-    Average / minimum / maximum beam momenta normalized by reference particle momentum (unit: dimensionless, radians for transverse momenta)
+    Average / minimum / maximum beam momentum deviation from the reference momenta and normalized by the longitudinal reference particle momentum (unit: dimensionless, radians for transverse momenta)
```

Comment:
```suggestion
    Average / minimum / maximum beam momentum deviation from the reference particle momentum and normalized by the magnitude of the reference particle momentum (unit: dimensionless, radians for transverse momenta)
```

Reply:
While we refer to "beam particles" vs. "reference particle", using the word "beam" here could lead to confusion.  I think the meaning is now clear without it.

Reply:
Good point, thank you Chad! :slightly_smiling_face: